consumers/email-consumer/Dockerfile

```
@@ -1,16 +1,35 @@
-    pip install -r /tmp/requirements.txt && \
-    adduser --disabled-password --no-create-home codesirius && \
-    mkdir -p /app && \
-    chown -R codesirius /app
+# add poetry to PATH
+ENV PATH="${PATH}:${POETRY_HOME}/bin"
+
+# create a non-root user
+RUN adduser --disabled-password codesirius
+
+# install poetry
+RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)